lib/rollup-plugin-less.js

```diff
@@ -0,0 +1,47 @@
+import less from 'less';
+import { createFilter } from 'rollup-pluginutils';
+
+function renderSync (code, options) {
+    return less
+        .render(code, options)
+        .then(function (output) {
+            return output;
+        }, function (err) {
+            throw err;
+        });
+}//renderSync
+
+export default function plugin (options={}) {
+    let _defaults = {
+        include: [ '**/*.less', '**/*.css' ],
+        exclude: [ 'node_modules/**' ],
+        options: {}
+    };
+    let cfg = Object.assign({}, _defaults, options);
+
+    const filter = createFilter(cfg.include, cfg.exclude);
+
+    return {
+        name: 'less',
+        transform: async function transform (code, fileName) {
+            if (!filter(fileName)) {
+                return null;
+            }
+
+            cfg.options.filename = fileName;
+
+            try {
+                let rendered = await renderSync(code, cfg.options);
+
+                return {
+                    code: `export default ${JSON.stringify(rendered.css)};`,
+                    map: {
+                        mappings: rendered.map
+                    }
+                };
+            } catch (error) {
+                throw error;
+            }
+        }
+    };
+};
```

Comment:
It's great we were able to get the exact functionality we need - While this is good for now, I think in the future it would be good to have this be it's own `rollup-plugin-inline-less` plugin that can exist on it's own. That way it can be imported and consumed exactly like the other rollup plugins and shared with others.